### PR TITLE
review 4.1.2

### PR DESCRIPTION
**Pull request by jordimassaguerpla**

### CODE REVIEW

File: adoc/MAIN.release-notes.adoc [hunk 1]
```diff
@@ -32,7 +32,10 @@ Please note that {productname} deployment on AWS may not be functionally complet
 {productname} can now be deployed with *{tf} 0.12*. All details of the new version
 can be found in the link:https://www.hashicorp.com/blog/terraform-0-1-2-preview/[HashiCorp Documentation].
 The official website for the {tf} 0.12 upgrade is https://www.terraform.io/upgrade-guides/0-12.html.
-Add CLI etcdctl to provide access to etcd container.
+
+=== Add CLI etcdctl
+
+You can now install etcdctl, the CLI to provide access to etcd container from the management host.
```
Comment (r0ckarong) [inline adoc/MAIN.release-notes.adoc]: Please replace the string with a formatted one

```
You can now install `etcdctl`, the CLI to provide access to the etcd container from the management host.
```

File: adoc/MAIN.release-notes.adoc [hunk 2]
```diff
@@ -109,6 +112,19 @@ SUSE update the ``user_data``. This should be removed as soon as possible after
 migration to {tf} 0.12.
 =========
 
+==== etcdctl
+Run `zypper in etcdctl` in the management host to install etcdctl.
+
+==== Update packages for general fixes
+Update skuba package and patterns-caasp-Management on your management workstation as you would do with any other package.
```
Comment (r0ckarong) [inline adoc/MAIN.release-notes.adoc]: Same here, please put in backtick quotes for monospace formatting of package names.

File: adoc/MAIN.release-notes.adoc [hunk 3]
```diff
@@ -109,6 +112,19 @@ SUSE update the ``user_data``. This should be removed as soon as possible after
...
+
+Refer to: link:https://documentation.suse.com/sles/15-SP1/single-html/SLES-admin/#sec-zypper-softup-update[https://documentation.suse.com/sles/15-SP1/single-html/SLES-admin/#sec-zypper-softup-update]
+
+Updating patterns-caasp-Management will install the new terraform providers for AWS.
```
Comment (r0ckarong) [inline adoc/MAIN.release-notes.adoc]: Again, please format pattern name.

**Review by r0ckarong — CHANGES_REQUESTED** [review]

**Comment by jordimassaguerpla** [comment]
@r0ckarong I just fixed the package names with `

**Comment by r0ckarong** [comment]
Waiting for Travis to validate then I will merge. I think the other PR should be about finished now as well.

**Pull request merged**